base/bucket_test.go

```diff
@@ -231,9 +230,49 @@ func TestGetStatsVbSeqno(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(ts *testing.T) {
 			actualUUIDs, actualHighSeqnos, err := GetStatsVbSeqno(test.stats, maxVbno, false)
-			goassert.Equals(ts, err, nil)
-			goassert.DeepEquals(ts, actualUUIDs, test.expectedUUIDs)
-			goassert.DeepEquals(ts, actualHighSeqnos, test.expectedHighSeqnos)
+			assert.NoError(ts, err)
+			assert.True(t, reflect.DeepEqual(actualUUIDs, test.expectedUUIDs))
```

Comment:
assert.Equal from testify does a deep equal

```suggestion
			assert.Equal(t, test.expectedUUIDs, actualUUIDs)
```

Reply:
Makes more sense; this is fixed in the latest revision.